tutorials/btp-cockpit-setup/btp-cockpit-setup.md

```diff
@@ -1,7 +1,9 @@
 ---
 parser: v2
+author_name: Annika Gonnermann
+author_ profile: https://github.com/AnnikaGonnermann
 time: 5
-tags: [ tutorial>beginner, software-product>sap-business-technology-platform, tutorial>type>free-tier]
+tags: [ tutorial>beginner; software-product>sap-business-technology-platform, tutorial>type>free-tier]

```

Comment:
Why did you replace the comma with a semicolon?

Reply:
I saw this at another tutorial but Michael already told me that it doesn't make sense. Tried to change it, so is it now fine?